src/lib/operators/aggregate_hash.cpp

```diff
@@ -475,8 +478,8 @@ std::shared_ptr<const Table> AggregateHash::_on_execute() {
   for (const auto& aggregate : _aggregates) {
     const auto column = aggregate.column;
 
-    // Output column for COUNT(*). int is chosen arbitrarily.
-    const auto data_type = !column ? DataType::Int : input_table->column_data_type(*column);
+    // Output column for COUNT(*). Int is chosen arbitrarily.
```

Comment:
Mmh, so that was wrong before, too?

Reply:
Do you mean the spelling? I chose `Int` because it's closer to `DataType::Int`.